### PR TITLE
Fix #2949: add diagnostics to navbar

### LINKED ISSUE #2949: Create a Administration->System->Diagnostics gui
Migrate the stuff from Administration->Clinic->Calendar->'Test System'
to a new nicer looking gui at:
Administration->System->Diagnostics

### PR DESCRIPTION
**Pull request by stu01509**
Fixes #2949 

#### Short description of what this resolves:
Add diagnostics in navbar Administration->System

image

### CODE REVIEW

**Comment by bradymiller** [comment]
hi @stu01509 , The link looks good. Now recommend creating a whole new page for this information (ie. remove it from smarty stuff). This will be the seed interface for a bunch of other Diagnostic tests in the future.

**Comment by stu01509** [comment]
Hi @bradymiller 

Did you mean remove Calendar from Administrator > Clinic?

**Comment by bradymiller** [comment]
Hi @stu01509 ,

This page will no longer have the `Test System` link in it:
image

Then the new page you create from Administration->System->Diagnostic will run the same stuff as that Test System page did (check out the code in Smarty that does this; pretty much gonna just be a copy/paste from that); the reason why moving it outside smarty into it's own page, is that will then begin to add more Diagnostic tests to that page which will be way easier to do outside of smarty.

**Comment by stu01509** [comment]
Hi @bradymiller 

OK, I got it, Do you mind I create another issue and PR to do this? and help me bring this PR into codebase?

File: interface/main/tabs/menu/menus/standard.json [hunk 1]
```diff
@@ -1085,6 +1085,18 @@
...
+          {
+            "label": "Diagnostics",
+            "menu_id": "adm0",
+            "target": "adm",
+            "url": "/interface/main/calendar/index.php?module=PostCalendar&type=admin&func=testSystem",
+            "children": [],
+            "requirement": 0,
+            "acl_req": [
+              "admin",
+              "users"
```
Comment (bradymiller) [inline interface/main/tabs/menu/menus/standard.json]: change `users` to `super`

**Comment by bradymiller** [comment]
hi @stu01509 , sure thing. just need one change to give it more strict permissions.

**Comment by stu01509** [comment]
Hi @bradymiller 

Please review it again.

**Comment by bradymiller** [comment]
hi @stu01509 , Looks good and bringing into codebase. I'll make a quick another issue to migrate. Thanks for the improvement! -brady

**Comment by bradymiller** [comment]
just realized, /i forgot to tel you bring this over to sites/default/documents/custom_menus/Custom.json . I'll bring in a quick commit to do that.

**Comment by stu01509** [comment]
> just realized, I forgot to tell you bring this over to sites/default/documents/custom_menus/Custom.json . I'll bring in a quick commit to do that.

Thanks for your noticed. 😃

**Pull request merged**